### ISSUE TITLE
[Durotar][Quest]The Admiral's Orders

### ISSUE THREAD

**Issue by xnemeyethx** [dalaranwow/dalaran-wow#5660]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
This is a quest aquired in Durotar after beating Benedict and looting his key from him. Thanks to this item we may open Benedict's Chest which contains a letter that gives the quest. This quest works perfectly… but only once. This is my theory but I think that when one character looted the chest another character will not loot this item. The only way to aquire it is to wait for server restart or crash.
**Expected behaviour**: Tell us what should happen instead
The letter - 100% drop, always ( if you have the key ofc ).
**Steps to reproduce the problem**:

1. Kill Benedict in Durotar and loot his key.
2. Open the chest and loot the letter.
3. Relog on another char and try to repeat steps 1 and 2.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Chest - https://wotlk.evowow.com/?object=3239
Npc - https://wotlk.evowow.com/?npc=3192

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by xnemeyethx** [comment]
I did this quest a few Times now but sometimes I opened the chest sth like 20 Times... no drop.

**Comment by Lothloryen** [comment]
#2690
